crates/biome_service/src/file_handlers/graphql.rs

```diff
@@ -529,6 +529,9 @@ pub(crate) fn fix_all(params: FixAllParams) -> Result<FixFileResult, WorkspaceEr
                             return ControlFlow::Break(action);
                         }
                     }
+                    FixFileMode::ApplySuppressions => {
+                        println!("No-op'ing in JSON.")
+                    }
```

Comment:
TODO: It looks to me like each language has the same code in these blocks, so I'd imagine I can bring the code from `javascript.rs`? (assuming that code is right)

Reply:
It's possible to make the code DRY, but be aware of the formatting step:

https://github.com/biomejs/biome/blob/7e29f06930e977d5f2165260ca87c21f414cf62e/crates/biome_service/src/file_handlers/javascript.rs#L720-L728

The `format_node` function is language specific. This is a nice to have, so we could DRY it later.